src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Speech.csproj">
+      <SupportedFramework>netcoreapp2.1;net45;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
```suggestion
      <SupportedFramework>netcoreapp2.0;net45;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```
switch to use netcoreapp2.0 in order to fix your build issues.

Reply:
thanks @joperezr